mne/connectivity/tests/test_utils.py

```diff
@@ -21,3 +23,31 @@ def test_indices():
                 assert np.sum(indices[0] == seed) == n_targets
             for target in targets:
                 assert np.sum(indices[1] == target) == n_seeds
+
+
+def test_degree():
+    """Test degree function."""
+    # degenerate conditions
+    with pytest.raises(ValueError, match='threshold'):
+        degree(np.eye(3), 2.)
+    # a simple one
+    corr = np.eye(10)
+    assert_array_equal(degree(corr), np.zeros(10))
+    # more interesting
+    corr = np.array([[0.5, 0.7, 0.4],
+                     [0.1, 0.3, 0.6],
+                     [0.2, 0.8, 0.9]])
+    deg = degree(corr, 1)
+    assert_array_equal(deg, [2, 2, 2])
+    # wanted values obtained with:
+    # import bct
+    # want = bct.degrees_und(bct.utils.threshold_proportional(corr, 0.25) > 0)
```

Comment:
???

Reply:
Leftover from Python version of brain connectivity toolbox Sheraz and I used before.

Reply:
I updated the comment to clarify